routing/routing.md

```diff
@@ -0,0 +1,60 @@
+# Routing
+
+This document depicts important information about configuring routing for an existing WSO2 product deployment in a
+ container platform.
+ 
+The following will be discussed in detail in the document.
+
+* [Exposing your application to outside](#exposing-your-application-to-outside )
+* [Configuring hostname](#configuring-hostname)
+* [DNS mapping](#dns-mapping)
+* [Configuring SSL](#configuring-ssl)
+* [Tips on Session Affinity](#tips-on-session-affinity)
+
+### Exposing your application to outside
+
+We recommend the use of [official WSO2 product Helm Charts](https://hub.helm.sh/charts/wso2) for production grade WSO2 
+product deployments in Kubernetes environments. [Kubernetes Ingress resources](https://kubernetes.io/docs/concepts/services-networking/ingress/)
+are used to expose expose WSO2 product services to outside of the Kubernetes environment.
+
+Kubernetes Ingress resources are capable of determining traffic based on the host and path forwarded to them by
+backends. It also operates on the application layer of the network stack (HTTP) and can provide features like cookie-based
+session affinity and SSL terminations. 
+
+>WSO2 recommend [Community NGINX](https://kubernetes.github.io/ingress-nginx/) as the Ingress controller for production 
+>grade product deployments in Kubernetes environments.
+
+### Configuring hostname
+
+For each WSO2 product that is exposed via the Ingress, must have a hostname. The hostname should be configured for
+each Ingress resource and for each WSO2 product configuration.
+
+### DNS mapping
+
+In order to access the services that are exposed through the Ingress controller, it is required to add DNS mappings
+ where each configured host name should be resolved to the Load balancer that fronts the Ingress controller. For
+testing purposes, it is possible to add host name mapping entries to the hosts file in the Operating system(For
+ example, Ubuntu: /etc/hosts).
```

Comment:
@ThilinaManamgoda a trivial suggestion. When defining a term like a file path shall we use code fragments (e.g. like `/etc/hosts` using back ticks)?